packages/core/lib/Kopflos.ts

```diff
@@ -148,6 +150,10 @@ export default class Impl implements Kopflos {
     this.start = onetime(async function (this: Impl) {
       await Promise.all(this.plugins.map(plugin => plugin.onStart?.()))
     }).bind(this)
+
+    this.ready = onetime(async function (this: Impl) {
+      await Promise.all(this.plugins.map(plugin => plugin.onReady?.()))
```

Comment:
do we need to `await` the promises here? is the server really ready to serve requests until the plugins finish their `onReady` task?

Reply:
True, technically, this `await` is irrelevant here because it's not awaited when the CLI starts:
https://github.com/zazuko/kopflos/blob/e82e9b5d64f9215a13f6e567cd28d2a40012a4dd/packages/cli/lib/command/serve.ts#L50-L53

And yes, the server should be able to serve requests. Of course some may still depend on the code in `onReady` hooks...